src/Microsoft.Health.Dicom.SqlServer/Registration/DicomFunctionsBuilderSqlServerRegistrationExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Health.Dicom.Core.Registration;
+using Microsoft.Health.Dicom.SqlServer.Features.Indexing;
+using Microsoft.Health.Dicom.SqlServer.Features.Retrieve;
+using Microsoft.Health.Dicom.SqlServer.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+using Microsoft.Health.SqlServer.Features.Storage;
+using Microsoft.Health.SqlServer.Features.Client;
+using Microsoft.Health.SqlServer;
+using Microsoft.Health.SqlServer.Features.Schema;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.SqlServer.Features.Schema.Manager;
+using Microsoft.Health.SqlServer.Configs;
+using Microsoft.Health.Dicom.Core.Extensions;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class DicomFunctionsBuilderSqlServerRegistrationExtensions
+    {
+        public static IDicomFunctionsBuilder AddSqlServer(
```

Comment:
These look duplicate. Is there a opportunity to create common SQL component extensions that we can reuse in both places?

Reply:
Working on...

Reply:
I put both extension method (`AddSqlServer`) into same class `ServiceCollectionRegistrationExtensions`, so that can easily extract common part.
So far I don't find much things in common. As we add more and more into Azure Functions, that could change.
I'm thinking of keep refactoring as we go,  how do you think?